projects/packages/publicize/src/js/classic-editor-connections.js

```diff
@@ -37,11 +37,11 @@ const validateFeaturedMedia = ( $, connectionsNeedValidation ) => {
 
 	const connectionNeedsMediaString = isEnhancedPublishingEnabled
 		? /* translators: %s is the link to the media upload best practices. */ __(
-				'You need a valid image or video in your post to share to Instagram. <a href="%s" rel="noopener noreferrer" target="_blank">Learn more</a>.',
+				'You need a featured image to share to Instagram - or use Gutenberg editor for more advanced media features! <a href="%s" rel="noopener noreferrer" target="_blank">Learn more</a>.',
```

Comment:
Is the '-' there on purpose? 

Should we say "You need a featured image to share to Instagram. Use the block editor for more advanced media features"

Reply:
We should refer to it as 'the block editor' rather than Gutenberg.

Reply:
Updated!